src/app/api/statistics/route.js

```diff
@@ -17,6 +18,14 @@ export const GET = async () => {
   }
 
   try {
+    const statisticsDocRef = doc(db, "statistics", "statistics");
+    const eventsCollectionRef = collection(db, "events");
+
+    const [statisticsDoc, eventsSnapshot] = await Promise.all([
```

Comment:
keep the same naming for these, u can call both a snapshot or just leave it at statistics and events